apps/remix-ide/src/app/plugins/parser/services/code-parser-gas-service.ts

```diff
@@ -40,7 +41,8 @@ export default class CodeParserGasService {
             return
         }
         this.plugin.currentFile = await this.plugin.call('fileManager', 'file')
-        this.plugin.nodeIndex.nodesPerFile[this.plugin.currentFile] = await this.plugin._extractFileNodes(this.plugin.currentFile, this.plugin.compilerAbstract)
+        // cast from the remix-plugin interface to the solidity one. Should be fixed when remix-plugin move to the remix-project repository
+        this.plugin.nodeIndex.nodesPerFile[this.plugin.currentFile] = await this.plugin._extractFileNodes(this.plugin.currentFile, this.plugin.compilerAbstract as unknown as lastCompilationResult)
```

Comment:
is `as unknown as lastCompilationResult` needed?

Reply:
yes for the moment. see the comment above...